extras/ioc_scan_by_host.sh

```diff
@@ -50,15 +50,16 @@ echo Instances are: "${instances[*]}"
 # Loop through all instance IDs
 for instance_id in "${instances[@]}"; do
   echo
-  echo Searching "$instance_id":
+  echo -n Searching "$instance_id":
 
-  # Use find-grep to search for IOC strings in log files, ignoring
-  # *.journal files.  We pipe the result into another grep process
-  # that uses the --invert-match grep flag to exclude matches (e.g.,
-  # from sudo.log) that contain our grep command (e.g. sudo.log).
+  # Use find-zgrep to search for IOC strings in log files (including
+  # gzipped log files), ignoring *.journal files.  We pipe the result
+  # into another grep process that uses the --invert-match grep flag
+  # to exclude matches (e.g., from sudo.log) that contain our grep
```

Comment:
Remove extraneous comma (I know it predates this PR):
```suggestion
  # to exclude matches (e.g. from sudo.log) that contain our zgrep
```